### ISSUE TITLE
You can Use `cx` here

### ISSUE THREAD

**Issue by anubra266** [YannBirba/vps-panda-css#7]
You can Use `cx` here:

https://github.com/YannBirba/vps-panda-css/blob/04dda2554838da7a1f9765eff31b41a5cd2eda50/renderer/PageShell.tsx#L29
 
```js
import { css, cx } from '@/styled-system/css'
```
```jsx
<button
 className={cx(
   "panda-theme-toggle-button",
   css({
    position: "relative",
    /*...*/
   })
 )}
/>
```

**Comment by YannBirba** [comment]
Yeah that's right ! Thank you for your issue 🐼

**Comment by YannBirba** [comment]
Fixed in : [5ca8324](https://github.com/YannBirba/vps-panda-css/commit/5ca83248dee4fbe837c833340ed56eb283676b1a)

**Comment by YannBirba** [comment]
Fixed in : [5ca8324](https://github.com/YannBirba/vps-panda-css/commit/5ca83248dee4fbe837c833340ed56eb283676b1a)
